sdk/apimanagement/Azure.ResourceManager.ApiManagement/src/autorest.md

```diff
@@ -17,6 +17,10 @@ modelerfour:
   flatten-payloads: false
 skip-serialization-format-xml: true
 
+no-property-type-replacement: 
+- ApiManagementServiceIdentity
+- PortalSettingsContractData
```

Comment:
This configuration is not changing anything - `PortalSettingsContractData` is not being replaced by anything previously.

Reply:
Fxied